tests/ai_safety/guardrails/constants.py

```
@@ -132,3 +134,5 @@ class GuardrailsDetectionPrompt:
 nHtAnnHtAnnHtAnnHtAnnHtAnnHtAnnHtAnnHtAnnHtAnnHtAnnHtAnnHtAnnHtAn
 nHtAnnHtAnnHtAnnHtAg==
 -----END PRIVATE KEY-----"""  # pragma: allowlist secret
+
+MINIO_SECRET_KEY_VALUE = "supersecret"  # pragma: allowlist secret
```

Ruff (0.15.20)

[error] Possible hardcoded password assigned to: "MINIO_SECRET_KEY_VALUE"

(S105)